src/erlcloud_ddb2.erl

```diff
@@ -766,6 +776,29 @@ undynamize_expression(Expression, _) ->
 undynamize_expression_attribute_names(Names, _) ->
     Names.
 
+-spec undynamize_global_table_status(binary(), undynamize_opts()) -> table_status().
+undynamize_global_table_status(<<"CREATING">>, _) -> creating;
+undynamize_global_table_status(<<"UPDATING">>, _) -> updating;
+undynamize_global_table_status(<<"DELETING">>, _) -> deleting;
+undynamize_global_table_status(<<"ACTIVE">>, _)   -> active.
+
+-spec undynamize_replica_description(jsx:json_term(), undynamize_opts()) -> replica_description().
+undynamize_replica_description(ReplicaDescription, _) ->
+    #ddb2_replica_description{region_name = proplists:get_value(<<"RegionName">>, ReplicaDescription)}.
+
+-spec undynamize_replica_descriptions(jsx:json_term(), undynamize_opts()) -> [replica_description()].
+undynamize_replica_descriptions(ReplicaDescriptions, Opts) ->
+    [undynamize_replica_description(ReplicaDescription, Opts)
+     || ReplicaDescription <- ReplicaDescriptions].
+
+-spec undynamize_replica(jsx:json_term(), undynamize_opts()) -> replica().
+undynamize_replica(Replica, _) ->
```

Comment:
so sad these two c/p funs differ only in the actual record used...but this is outside of scope of the PR

Reply:
Yeah, I decided to shadow their difference for [Replica](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Replica.html) and [ReplicaDescription](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ReplicaDescription.html). They are symmetric at this time, but it appears to me that they may change in the future, in particular, I think ReplicaDescription will gain additional fields once they, er... finish it?